src/main/java/mat/server/FeatureFlagServiceImpl.java

```diff
@@ -24,6 +25,7 @@
 	FeatureFlagDAO featureFlagDAO;
 
 	@Override
+	@Cacheable("featureFlags")
```

Comment:
I think @adongare  told he needs it cached on dao level...

Reply:
I want just one flag and not map or list of flags. if we cache this at dao layer, and if another dao needs it, I still need to perform this processing to find if flag is on/off. I would prefer a new method which I added already to find if flag is on/off by keeping caching at service level.

Reply:
@adongare As far as I remember, FindFeatureFlagByName(String name) implementation is replaced with Hashmap.

Reply:
yeah.. actually DAO can be changed to return a MAP (I had originally proposed that in the code review before) and then it can be annotated as Cached.

Reply:
okay, if we move map preparation to dao and add caching at dao layer, that will work.